uint8 id
---
uint8 operatingMode
int32 result
